generated/provider_dependencies.json

```diff
@@ -343,7 +343,7 @@
       "google-cloud-dataplex>=0.1.0",
       "google-cloud-dataproc-metastore>=1.2.0,<2.0.0",
       "google-cloud-dataproc>=3.1.0",
-      "google-cloud-dlp>=0.11.0,<2.0.0",
+      "google-cloud-dlp>=3.0.0,<4.0.0",
```

Comment:
Could you please describe in the comment why we (still) upper bound it - following the rules we adopted recently ? (I know we do not have it for other cases but might be a good idea to add similar comments to others):

https://github.com/apache/airflow#approach-to-dependencies-of-airflow

> Whenever we upper-bound such a dependency, we should always comment why we are doing it - i.e. we should have a good reason why dependency is upper-bound. And we should also mention what is the condition to remove the binding.

Reply:
@potiuk no reason why I added this, only to be more safety in case where we will have big changes in the SDK and operators will stop working.

Reply:
@potiuk I checked once again and we need upper-bound restriction, the reason is the same as we have in others conflicts in the dependencies regarding `protobuf` and `proto-plus` packages